modules/import_har.py

```diff
@@ -35,50 +39,55 @@ def run(argv=None):
 
     # TODO add metric counters for files in, processed, written to GCP & BQ
 
-    with beam.Pipeline(options=pipeline_options) as p:
-        parsed = (
-            p
-            | ReadHarFiles(known_args.subscription, known_args.input)
-            | "ParseHar" >> beam.ParDo(ImportHarJson()).with_outputs("page", "requests")
-        )
-        pages, requests = parsed
-        requests = requests | "FlattenRequests" >> beam.FlatMap(
-            lambda elements: elements
-        )
+    p = beam.Pipeline(options=pipeline_options)
```

Comment:
Variable assignment instead of using `with` (Context Manager) since its `__exit__()` method uses `wait_until_finish()`, causing blocking until completion from the client/launch process.

https://github.com/apache/beam/blob/93fd177d43f42a6d92cc8825e058594d870ad449/sdks/python/apache_beam/pipeline.py#L597